3-reveal/migrations/5-IRS/3-Namibia-2019/deploy/redo_dependencies_of_namibia_irs_structures.psql

```diff
@@ -0,0 +1,350 @@
+-- Deploy reveal_irs_namibia_2019:redo_dependencies_of_namibia_irs_structures to pg
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+-- namibia_irs_export --
+DROP MATERIALIZED VIEW IF EXISTS pending_:schema .namibia_irs_export CASCADE;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS pending_:schema .namibia_irs_export
```

Comment:
I wonder how hard it would be to write a PSQL recreate_matview function to automate some of this...

Reply:
I added an issue to try and track this https://github.com/OpenSRP/opensrp-reveal-datawarehouse/issues/78